CMakeLists.txt

```diff
@@ -49,6 +55,7 @@ include (DetermineGflagsNamespace)
 set (CMAKE_THREAD_PREFER_PTHREAD 1)
 
 if (WITH_GFLAGS)
+  hunter_add_package (gflags)
   find_package (gflags)
```

Comment:
Check the usage:
* https://github.com/ruslo/hunter/wiki/pkg.gflags

We need `find_package(gflags CONFIG REQUIRED)`